src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.php

```diff
@@ -0,0 +1,222 @@
+            ->abstract()
+            ->tag('cache.pool', ['clearer' => 'cache.default_clearer', 'reset' => 'reset'])
+            ->tag('monolog.logger', ['channel' => 'cache'])
+            ->args([
+                '', // namespace
+                0, // default lifetime
+                '', // version
+            ])
+            ->call('setLogger', [service('logger')->ignoreOnInvalid()])
+
+        ->set('cache.adapter.doctrine', DoctrineAdapter::class)
+            ->abstract()
+            ->tag('cache.pool', [
+                'provider' => 'cache.default_doctrine_provider',
+                'clearer' => 'cache.default_clearer',
+                'reset' => 'reset',
+                ])
+            ->tag('monolog.logger', ['channel' => 'cache'])
+            ->args([
+                '', // Doctrine provider service
+                '', // namespace
+                0, // default lifetime
+            ])
+            ->call('setLogger', [service('logger')->ignoreOnInvalid()])
+
+        ->set('cache.adapter.filesystem', FilesystemAdapter::class)
+            ->abstract()
+            ->tag('cache.pool', ['clearer' => 'cache.default_clearer', 'reset' => 'reset'])
+            ->tag('monolog.logger', ['channel' => 'cache'])
+            ->args([
+                '', // namespace
+                0, // default lifetime
+                '%kernel.cache_dir%/pools',
+                service('cache.default_marshaller')->ignoreOnInvalid(),
+            ])
+            ->call('setLogger', [service('logger')->ignoreOnInvalid()])
+
+        ->set('cache.adapter.psr6', ProxyAdapter::class)
+            ->abstract()
+            ->tag('cache.pool', [
+                'provider' => 'cache.default_psr6_provider',
+                'clearer' => 'cache.default_clearer',
+                'reset' => 'reset',
+                ])
+            ->args([
+                '', // PSR-6 provider service
+                '', // namespace
+                0, // default lifetime
+            ])
+
+        ->set('cache.adapter.redis', RedisAdapter::class)
+            ->abstract()
+            ->tag('cache.pool', [
+                'provider' => 'cache.default_redis_provider',
+                'clearer' => 'cache.default_clearer',
+                'reset' => 'reset',
+            ])
+            ->tag('monolog.logger', ['channel' => 'cache'])
+            ->args([
+                '', // Redis connection service
+                '', // namespace
+                0, // default lifetime
+                service('cache.default_marshaller')->ignoreOnInvalid(),
+            ])
+            ->call('setLogger', [service('logger')->ignoreOnInvalid()])
+
+        ->set('cache.adapter.memcached', MemcachedAdapter::class)
+            ->abstract()
+            ->tag('cache.pool', [
+                'provider' => 'cache.default_memcached_provider',
+                'clearer' => 'cache.default_clearer',
+                'reset' => 'reset',
+                ])
```

Comment:
This line should remove an indent level (it has to be at the same level as `->tag(`)

Reply:
done